### ISSUE TITLE
Being an "Owner" is not sufficient for installation

### ISSUE THREAD

**Issue by mfehlhaber** [gitpod-io/gitpod-microsoft-aks-guide#11]
### Bug description

Running `make install` on a subscription where I am "Owner" led to an incomplete installation and the following output:

> Allow Kubernetes managed identity to access the storage account...
> Insufficient privileges to complete the operation.

I am lacking some right regarding identities in AAD. Assigning myself the role "Managed Identity Contributor" did not help. So unless you can find a way to make it work without, there seems to be some precondition missing.

### Steps to reproduce

`make install` as Owner

**Comment by MrSimonEmms** [comment]
@mfehlhaber can you try this with the latest version in `main` please? I think I've fixed that

tl;dr - it was because we were creating and assigning AAD roles which exists outside of the normal RBAC process
